test/e2e/service-request.spec.ts

```diff
@@ -189,7 +189,7 @@ describe('Service Request Pallet Integration Tests', () => {
     order = await orderPromise;
 
     const promise: Promise<ServiceInvoice> = new Promise((resolve, reject) => { // eslint-disable-line
-      processRequest(api, pair, lab.accountId, serviceRequest.hash, order.id, order.dnaSampleTrackingId, stakingAmount.toString(), () => {
+      processRequest(api, pair, lab.accountId, serviceRequest.hash, order.id, order.dnaSampleTrackingId, () => {
         queryServiceInvoiceById(api, serviceRequest.hash)
```

Comment:
Thought this has been remove ? Please also remove `queryServiceInvoiceById` & `queryServiceInvoiceByOrderId` on e2e test as well.

Reply:
done